### PR TITLE
Website fixes post 1.3.1 and 1.2.3

### PR DESCRIPTION
**Pull request by TimMoore**
This might be easier to review commit-by-commit: I did a lot of reformatting, but I did all of the reformatting in separate commits from the other substantive changes.

### CODE REVIEW

File: src/main/twirl/com/lightbend/lagom/docs/getstartedjava.scala.html [hunk 1]
```diff
@@ -79,66 +64,93 @@
...
+                                    my-first-project.
+                                </li>
+                                <li>
+                                    <code>version</code> &#8212; the version for your project, press <b>Enter</b> to
+                                    accept the default.
+                                </li>
+                                <li>
+                                    <code>package</code> &#8212; defaults to the same value as the <code>groupId</code>
+                                </li>
+                            </ul>
```
Comment (ignasi35) [inline src/main/twirl/com/lightbend/lagom/docs/getstartedjava.scala.html]: We have to review the lists' styles. This seems to be using a _not great_ padding: 

screen_shot_2017-03-07_at_14_27_29

See also #32

Comment (ignasi35) [inline src/main/twirl/com/lightbend/lagom/docs/getstartedjava.scala.html]: I wouldn't do anything in this PR. Just added the comment to x-ref from #32

File: src/main/twirl/com/lightbend/lagom/docs/getstartedjava.scala.html [hunk 2]
```diff
@@ -15,58 +15,43 @@
...
-                  
-                     </div>
-                     
-                     
-                    
-                     
-                     <div class="small-12 columns">
+                        (If you aren’t using one of these tools already, see this information to
+                        help you choose between <a href="http://www.scala-sbt.org/0.13/docs/index.html">sbt</a> and
+                        <a href="https://maven.apache.org">Maven</a>.) The Lagom development environment runs on
```
Comment (ignasi35) [inline src/main/twirl/com/lightbend/lagom/docs/getstartedjava.scala.html]: Should the `.` be outside the parens in the sinppet

````
<a href="https://maven.apache.org">Maven</a>.) 
``` 
??

PS: I read the comments wrt UK/US/AUS variants and can't remember if this was discussed there. Feel free to ignore.

File: src/main/twirl/com/lightbend/lagom/docs/getstartedjava.scala.html (hunk 1 above)
Comment (TimMoore) [inline src/main/twirl/com/lightbend/lagom/docs/getstartedjava.scala.html]: Sounds good

File: src/main/twirl/com/lightbend/lagom/docs/getstartedjava.scala.html (hunk 2 above)
Comment (TimMoore) [inline src/main/twirl/com/lightbend/lagom/docs/getstartedjava.scala.html]: I think when the complete sentence is parenthesized, the period goes inside.

**Comment by TimMoore** [comment]
Sorry, @rstento but I'm not sure which part you're referring to. Could you comment on the specific line? I did test everything to make sure that the text matched up with what was happening, but maybe I missed something.

**Pull request merged**